test/reductions/thomasvandoren/sorted.chpl

```diff
@@ -3,7 +3,8 @@
  * or not.
  */
 
-public use Sort;
+public use Sort except isSorted;
```

Comment:
Unless I made a mistake, I think that this `public use` isn't required either.  That said, I don't object to this change either per se (but relying on the `private use` is how I'd expect people to write the code in practice, so it might put the weight on the right foot).

Reply:
That approach on its own doesn't solve this one. This particular file `sorted.chpl` is `.notest`'d and it is tested in `test/TestSorted.chpl` which begins with a `use sorted`. If we were to adjust `sorted.chpl` to have `private use Sort` (or just `use Sort`) then in `test/TestSorted.chpl` we will have an "unresolved call" for the `sort` call it makes on line 15. At that point, if we simply add `use Sort;` to `test/TestSorted.chpl`, then we get multiply-defined error for `isSorted` (since `isSorted` is available both as `Sort.isSorted` and as `TestSorted.isSorted`). To fix that, we would have to add `use Sort except isSorted;` in `test/TestSorted.chpl`.

I view the change already this PR as better but I am open to doing what is described above.

Reply:
OK, thanks for the explanation!